w3af/plugins/audit/cors_origin.py

```diff
@@ -167,7 +167,14 @@ def _allow_methods(self, forged_req, url, origin, response,
             msg %= (url, ACCESS_CONTROL_ALLOW_METHODS,
                     allow_methods, ', '.join(report_sensitive))
 
-            v = Vuln(name, msg, severity.LOW, response.get_id(), self.get_name())
+            v = Vuln(
+                name,
+                msg,
+                severity.LOW,
+                response.get_id(),
+                self.get_name(),
+                vulndb_id=10010,
```

Comment:
Would it be possible for Holm to contribute back to the `vulndb`? If we do that, all the w3af users would get these detailed descriptions for the vulnerabilities.

Reply:
I brought up the discussion internally in Holm.

Reply:
And what was the resolution?

Reply:
So far there's no decision to contribute to `vulndb`. Holm holds off